src/components/Header.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Calculator.css';
+
```

Comment:
- [ ] There seems to be a problem with this import right here as the name of the file is `calculator.js` and not `Calculator.js`. Please modify the import to match the actual file name in order to get rid of the error below 👇 
![image](https://user-images.githubusercontent.com/46839548/190829613-87946dc8-6215-49d1-ad7e-d8bfc45ecd0a.png)